docs/guests.md

```diff
@@ -293,6 +293,19 @@ Help is welcome to help us reconcile both procedures into one.
 
 **Note**: Also have a look at our [Troubleshooting Guide - Windows PV-Tools](troubleshooting.html#windows-agent-pv-tools).
 
+### VMs with INACCESSIBLE_BOOT_DEVICE error
+
+You can try to manually inject the missing drivers in recovery mode.
+
+* Get the "Drivers" folder from the XCP Tools installation path (C:\PROGRAM FILES...) - from another VM or install the tools somewhere else to get it.
```

Comment:
Could it work directly by using the ISO available on download from https://github.com/xcp-ng/win-pv-drivers ?

Eg https://github.com/xcp-ng/win-pv-drivers/releases/download/v8.2.2.200-RC1/XCP-ng.Windows.PV.Drivers.8.2.2.200-rc1.iso

Reply:
The ISOs contain the Installer for x86 and x64 but we need the driver files like xenvbd.inf and xenvbd.sys - so its more easy to just get them from the installer or another installation ... i had a quick look but it seems the driver files are not to be found precompiled in the win-xenvbd repo.

Reply:
Okay so let's keep it that way then :)